cmd/config-dir.go

```diff
@@ -102,7 +102,7 @@ func mustGetDefaultConfigDir() string {
 	return filepath.Join(homeDir, defaultMinioConfigDir)
 }
 
-var configDir = &ConfigDir{dir: mustGetDefaultConfigDir()}
+var configDir = &ConfigDir{}
```

Comment:
If you don't need default value, why do you need to init empty value?

Reply:
This comment is still valid

Reply:
done

Reply:
`var configDir = &ConfigDir{dir: getDefaultConfigDir()}` makes `configDir` initialized properly